.mci.yml

```diff
@@ -832,45 +842,48 @@ tasks:
     - name: clang-tidy
       commands:
         - func: "setup"
-        - func: "install_c_driver"
+        - func: "fetch_latest_cmake"
         - func: "compile"
         - func: "clang-tidy"
 
     - name: compile_and_test_with_shared_libs
       commands:
         - func: "setup"
         - func: "start_mongod"
-        - func: "install_c_driver"
+        - func: "fetch_latest_cmake"
         - func: "compile"
           vars:
               RUN_DISTCHECK: 1
         - func: "clone_drivers-evergreen-tools"
         - func: "run_kms_servers"
+        - func: "install_c_driver"
```

Comment:
The reason for including "install_c_driver" just before "test" is that the example projects want to link with `libbson-static-1.0`. Here's an example CI failure when not including the C driver install before building the examples: https://spruce.mongodb.com/task/cxx_driver_ubuntu1604_debug_std_experimental_compile_and_test_with_static_libs_extra_alignment_patch_cd9542cbf0186935f055e3e03469764033f70457_64c2982ae3c33127d03bebae_23_07_27_16_15_39/logs?execution=0

Reply:
The reason for "fetch_latest_cmake" is because many of our CI scripts are in the C driver repo. Do you have an alternative suggestion for fetching these scripts?

Reply:
OK. I was not sure what the purpose is of `fetch_latest_cmake`.